schema/source_schema.json

```diff
@@ -76,9 +76,84 @@
       "properties": {
         "country": {
           "type": "string"
+        },
+        "province": {
+          "type": "string"
+        },
+        "state": {
+          "type": "string"
+        },
+        "county": {
+          "type": "string"
+        },
+        "city": {
+          "type": "string"
+        },
+        "geometry": {
+          "description": "from https://github.com/fge/sample-json-schemas/blob/master/geojson/geometry.json",
+          "oneOf": [
```

Comment:
Instead of copying bits of the GeoJSON schema into our schema you can reference it like here: https://github.com/osmlab/editor-layer-index/blob/gh-pages/schema.json#L14